### PR TITLE
Python 3, style fixes from galaxy-lib.

### PR DESCRIPTION
**Pull request by jmchilton**
- Python 3 fix - https://github.com/galaxyproject/galaxy-lib/commit/07540b6ab56b3ced0bebe2e3979f84c545f8a0ab
- Python 3 fix - https://github.com/galaxyproject/galaxy-lib/commit/eea2b79c969881af053843082b9ebd58df96296f (thanks to @nsoranzo).
- Cleaner imports - https://github.com/galaxyproject/galaxy-lib/commit/fc6c3d620e0de1aa94b503cbb85385dbcdb3e761 (thanks to @nsoranzo).
- PEP-8 fix - https://github.com/galaxyproject/galaxy-lib/commit/1e90c7b6d659e4141f803d79947c772741c5bd23

### CODE REVIEW

File: lib/galaxy/web/stack/__init__.py [hunk 1]
```diff
@@ -210,8 +210,8 @@ def get_app_kwds(cls, config_section, app_name=None):
             config_file = uwsgi_opt.get("galaxy_config_file")
         # check for --yaml or --json uWSGI config options next
         if config_file is None:
-            config_file = (UWSGIApplicationStack._get_config_file(uwsgi_opt.get("yaml"), yaml.safe_load, config_section)
-                           or UWSGIApplicationStack._get_config_file(uwsgi_opt.get("json"), json.load, config_section))
+            config_file = (UWSGIApplicationStack._get_config_file(uwsgi_opt.get("yaml"), yaml.safe_load, config_section) or
+                           UWSGIApplicationStack._get_config_file(uwsgi_opt.get("json"), json.load, config_section))
```
Comment (nsoranzo) [inline lib/galaxy/web/stack/__init__.py:214]: We now ignore W503, see https://github.com/galaxyproject/galaxy/pull/2594

File: lib/galaxy/tools/verify/test_data.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def __init__(self, repository, environ):
         self.updated = False
         repo_cache = environ.get("GALAXY_TEST_DATA_REPO_CACHE", "test-data-cache")
         m = hashlib.md5()
-        m.update(repository)
+        m.update(repository.encode('utf-8'))
```
Comment (nsoranzo) [inline lib/galaxy/tools/verify/test_data.py:81]: See https://github.com/galaxyproject/galaxy-lib/pull/94#discussion_r170977017

File: lib/galaxy/tools/cwl/util.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import tempfile
 from collections import namedtuple
 
-from six import iteritems, python_2_unicode_compatible, StringIO
+from six import BytesIO, iteritems, python_2_unicode_compatible
```
Comment (nsoranzo) [inline lib/galaxy/tools/cwl/util.py:12]: These could be probably split on multiple lines.

File: lib/galaxy/web/stack/__init__.py (hunk 1 above)
Comment (jmchilton) [reply lib/galaxy/web/stack/__init__.py:214]: So we need to update galaxy-lib then - I adjusted this to pass tests there. Thanks for the pointer.

**Pull request closed without merging**